Makefile.common

```diff
@@ -1,14 +1,22 @@
 RSPDIR             = $(ROOT_DIR)/mupen64plus-rsp-hle
 CXD4DIR            = $(ROOT_DIR)/mupen64plus-rsp-cxd4
 CORE_DIR           = $(ROOT_DIR)/mupen64plus-core
-PNG_DIR            = $(ROOT_DIR)/custom/dependencies/libpng
-ZLIB_DIR           = $(ROOT_DIR)/custom/dependencies/libzlib
 LIBRETRO_COMM_DIR  = $(ROOT_DIR)/libretro-common
 RSPDIR_PARALLEL    = $(ROOT_DIR)/mupen64plus-rsp-paraLLEl
 AUDIO_LIBRETRO_DIR = $(ROOT_DIR)/custom/mupen64plus-core/plugin/audio_libretro
 VIDEODIR_GLIDEN64  = $(ROOT_DIR)/GLideN64
 VIDEODIR_ANGRYLION = $(ROOT_DIR)/mupen64plus-video-angrylion
 
+MINIZIP_DIR        = $(CORE_DIR)/subprojects/minizip
+PNG_DIR            = $(ROOT_DIR)/custom/dependencies/libpng
+XXHASH_DIR         = $(CORE_DIR)/subprojects/xxhash
+ZLIB_DIR           = $(ROOT_DIR)/custom/dependencies/libzlib
+
+SYSTEM_MINIZIP ?= 0
+SYSTEM_LIBPNG ?= 0
+SYSTEM_XXHASH ?= 0
+SYSTEM_ZLIB ?= 0
```

Comment:
I meant in `Makefile`, not in `Makefile.common`.
`Makefile.common` is not supposed to have any configuration vars like this, DYNAREC_USED is only a temp var and a exception.
After thats its good2go.

Reply:
Ah , sorry, should be good now.